shopify/product-reviews-template/.template/docs/setup.md

```diff
@@ -0,0 +1,9 @@
+# Getting started
+
+A list of steps that you should follow:
+
+1. [Connect to Shopify](https://docs.gadget.dev/guides/tutorials/connecting-to-shopify#connecting-to-shopify) using our built-in Shopify plugin.
```

Comment:
Can we also mention that protected customer data access is required?

Reply:
It seems like emails are also needed. We should call that out because it is its own box in the second section

Reply:
Good callouts